packages/compartment-mapper/test/fixtures-policy/node_modules/eve/README.md

```diff
@@ -0,0 +1 @@
+this package is called eve not mallory, because it's attempting to ensure that if identifying packages for the sake of policy enfocement is not tricked by the override `"dan": "hackity"` it specifies. Just in case we don't get a false negative from the test, both 'eve' and 'dan' are shorter names than the alternative path that's the correct one: `alice>hackity`
```

Comment:
Nits
```suggestion
This package is called `eve` not `mallory`, because it's attempting to ensure that if identifying packages for the sake of policy enfocement is not tricked by the override `"dan": "hackity"` it specifies. Just in case we don't get a false negative from the test, both `eve` and `dan` are shorter names than the alternative path that's the correct one: `alice>hackity`
```